docs/source/model-export/export-mnn-zipformer.rst

```diff
@@ -0,0 +1,188 @@
+.. _export_streaming_zipformer_transducer_models_to_mnn:
+
+Export streaming Zipformer transducer models to MNN
+----------------------------------------------------
+
+We use the pre-trained model from the following repository as an example:
+
+`<https://huggingface.co/pfluo/k2fsa-zipformer-bilingual-zh-en-t>`_ (backup link:
+`<https://huggingface.co/meixu/k2fsa-zipformer-bilingual-zh-en-t>`_)
+
+We will show you step by step how to export it to `MNN`_ and run it with `sherpa-MNN`_.
+
+.. hint::
+
+  We use ``Ubuntu 20.04``, ``torch 2.0.0``, and ``Python 3.8`` for testing.
+
+.. caution::
+
+  Please use a more recent version of PyTorch. For instance, ``torch 1.8``
+  may ``not`` work.
+
+1. Download the pre-trained model
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. hint::
+
+  You have to install `git-lfs`_ before you continue.
+
+
+.. code-block:: bash
+
+  cd egs/librispeech/ASR
+  git clone https://huggingface.co/meixu/k2fsa-zipformer-bilingual-zh-en-t
+
+  cd ..
+
+In the above code, we downloaded the pre-trained model into the directory
+``egs/librispeech/ASR/k2fsa-zipformer-bilingual-zh-en-t``.
+
+.. _export_for_mnn_install_mnn:
+
+2. Install MNN
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+  # We put MNN into $HOME/open-source/MNN
+  # You can change it to anywhere you like
+
+  cd $HOME
+  mkdir -p open-source
+  cd open-source
+
+  git clone https://github.com/alibaba/MNN
+  cd MNN
+  mkdir build && cd build
+
+  cmake \
+    -DMNN_BUILD_CONVERTER=ON \
+    -DMNN_BUILD_TORCH=ON \
+    -DMNN_BUILD_TOOLS=ON \
+    -DMNN_BUILD_BENCHMARK=ON \
+    -DMNN_EVALUATION=ON \
+    -DMNN_BUILD_DEMO=ON \
+    -DMNN_BUILD_TEST=ON \
+    -DMNN_BUILD_QUANTOOLS=ON
```

Comment:
```suggestion
    -DMNN_BUILD_QUANTOOLS=ON \
```

Please copy and paste the commands in the doc and ensure users can reproduce them locally.